R/SIRS_solve.R

```diff
@@ -18,17 +18,25 @@ equationsSIRS <- function(time, variables, parameters)
   list(c(dS, dI, dR, dN))
 }
 
-solveSIRS() <- function(beta = 0.001, gamma = 0.1, xi = 0.1, muB = 0, muD = 0, population = 500,
-                        susceptible = 499, infected = 1, recovered = 0,timesteps = 50)
-{
-  variables <- c(susceptible, infected, recovered, population,q)
-  names(variables) <- c("S", "I", "R", "N")
-  parameters <- c(beta, gamma, xi, muB, muD)
-  names(parameters) <- c("beta", "gamma", "xi", "muB", "muD")
-  
-  deSolve::lsoda(variables, seq(0, timesteps, by = 1),
-                 equationsSIR, parameters) |> as.data.frame()
-}
+solveSIRS <-
```

Comment:
Good catch here, I didn't notice that it was being assigned to a function previously.